Update description of Array.length property (#43981)
mdn/content: files/en-us/web/javascript/reference/global_objects/array/length/index.md

sidebar: jsref
---

> The **`length`** data property of an {{jsxref("Array")}} instance represents the number of slots in that array (this may be different to the number of elements as it may be a sparse array - with 'holes'). The value is an unsigned, 32-bit integer that is always numerically greater than the highest index in the array.

Reviewer: Generally okay but let's move it to the end

```suggestion
The **`length`** data property of an {{jsxref("Array")}} instance represents the number of slots in that array. The value is an unsigned, 32-bit integer that is always numerically greater than the highest index in the array. It may be greater than the number of elements if the array is [sparse](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array#array_methods_and_empty_slots).
```

Author: Looks good
